src/xdocs/config_blocks.xml

```diff
@@ -150,6 +150,34 @@ switch (a)
     <section name="EmptyBlock">
       <subsection name="Description">
         <p> Checks for empty blocks. This check does not validate sequential blocks. </p>
+
+        <p> Sequential blocks won't be checked. Also, no violations for fallthrough: </p>
+        <source>
+            switch (a) {
+            case 1:                          // no violation
+            case 2:                          // no violation
+            case 3: someMethod(); { }        // no violation
+            default: break;
+            }
+        </source>
+
+        <p> This check processes LITERAL_CASE and LITERAL_DEFAULT separately: </p>
+        <source>
+            switch (a) {
+            default:        // no violation for default as empty block belong to "case 1:"
+            case 1: { }     // violation
+            }
+        </source>
+        <p>
+            Do note that for the check to behave as shown above, LITERAL_CASE must be present under
+            tokens in the config file if a violation is desired for the empty block inside 'case'.
+            One possible configuration which would produce the above results is:
+        </p>
+        <source>
+&lt;module name=&quot;EmptyBlock&quot;&gt;
+    &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
+&lt;/module&gt;
+        </source>
```

Comment:
move description and configuration above result(code with violations).
Please remove LITERAL_CASE  from config to make it be representetive.